general/community/meetings.md

```diff
@@ -9,13 +9,14 @@ tags:
 
 Developer meetings are open to anyone interested in Moodle development.
 
-Next meeting: [Developer meeting April 2022](https://docs.moodle.org/dev/Developer_meeting_April_2022)
+Next meeting: [June 2022](./meetings/202206)
```

Comment:
You need to create the general/community/meetings/202206.md too and add the content for the June's meeting :-) You can duplicate general/community/meetings/202202.md and then adapt the content (I guess that will be the easier way for doing it because in that case, there is no page to migrate).

If a page has to be migrated (for instance, April's one), you can run the following command in your devdocs cloned project and then check that the content is OK:

```yarn migrate Developer_meeting_April_2022 /general/community/meetings/202204.md```

If you haven't downloaded/installed this project locally, let me know and I'll create these files for you :-)